### PR TITLE
Test pr 1835

### PR DESCRIPTION
**Pull request by sfc-gh-aalam**
Tests #1835

### CODE REVIEW

File: src/snowflake/connector/cursor.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 import uuid
 import warnings
 from enum import Enum
+from importlib import util
```
Comment (thomasaarholt) [inline src/snowflake/connector/cursor.py:18]: Looks like I didn’t remove the import that we decided was unnecessary. I’m used to Ruff removing these automatically on save in my projects, sorry about that!

Comment (sfc-gh-aalam) [reply src/snowflake/connector/cursor.py:18]: could you remove it and add a changelog entry. I'll merge and publish this again

Comment (thomasaarholt) [reply src/snowflake/connector/cursor.py:18]: I just removed it now, in #1835.

Happy new year!

**Review by sfc-gh-mkeller — APPROVED** [review]
Ohh, haha. This is from I was first learning about type-hints! 😅 

:ship: :ship:

**Pull request closed without merging**